src/components/inngang/inngangspanel.less

```diff
@@ -108,6 +108,13 @@
             margin: 1rem 0 0 44px;
         }
 
+        .inngangspanel__annullert {
```

Comment:
trenger å få til noe som ser bra ut på mobil her

Reply:
flytta denne i siste commit for å gjøre det pittelitt enklere å flytte på elementet inne i dokumentstrukturen, men det er fremdeles ikke fiksa